### PR TITLE
bump ios min version to 9

### PR DESCRIPTION
**Pull request by omegabytes**
I hit this error unexpectedly when attempting to publish `v2.0.0`:
```
pod trunk push Segment-Flurry.podspec --use-libraries --allow-warnings
Updating spec repo `trunk`

CocoaPods 1.9.1 is available.
To update use: `sudo gem install cocoapods`

For more information, see https://blog.cocoapods.org and the CHANGELOG for this version at https://github.com/CocoaPods/CocoaPods/releases/tag/1.9.1

Validating podspec
 -> Segment-Flurry (2.0.1)
    - ERROR | [iOS] unknown: Encountered an unknown error (CocoaPods could not find compatible versions for pod "Flurry-iOS-SDK":
  In Podfile:
    Segment-Flurry (from `/Users/alexander.gaesser/code/integrations-ios/analytics-ios-integration-flurry/Segment-Flurry.podspec`) was resolved to 2.0.1, which depends on
      Flurry-iOS-SDK (~> 10.2.0)

Specs satisfying the `Flurry-iOS-SDK (~> 10.2.0)` dependency were found, but they required a higher minimum deployment target.) during validation.

[!] The spec did not pass validation, due to 1 error.
```

Quick Google search tells me I need to bump iOS version. I've decided on ios 9 based on the data[ on this site](https://david-smith.org/iosversionstats/).
image

I set the example min target to v9 as well and reran example app tests which passed.

### CODE REVIEW

**Comment by omegabytes** [comment]
Test results:
```
2020-04-16 11:58:53.706660-0700 Segment-Flurry_Example[17607:5443022] <0x600002d11050:SEGSegmentIntegration, > Flushing 3 of 3 queued API calls.
2020-04-16 11:58:53.707412-0700 Segment-Flurry_Example[17607:5443022] Flushing batch {
    batch =     (
                {
            anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
            context =             {
                app =                 {
                    build = "1.0";
                    name = "Segment-Flurry_Example";
                    namespace = "org.cocoapods.demo.Segment-Flurry-Example";
                    version = "1.0";
                };
                device =                 {
                    id = "2B3659A2-DAD3-4E55-BC07-8A9D6E634CA3";
                    manufacturer = Apple;
                    model = "x86_64";
                    type = ios;
                };
                library =                 {
                    name = "analytics-ios";
                    version = "3.7.1";
                };
                locale = "en-US";
                network =                 {
                    cellular = 0;
                    wifi = 1;
                };
                os =                 {
                    name = iOS;
                    version = "13.4.1";
                };
                screen =                 {
                    height = 568;
                    width = 320;
                };
                timezone = "America/Los_Angeles";
                traits =                 {
                    anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
                    email = "tool@fake-segment-tester.com";
                    userId = "segment-fake-tester";
                };
            };
            integrations =             {
            };
            messageId = "A1915E4B-66DD-42E6-B9FC-3969A35E3551";
            timestamp = "2020-04-16T18:58:23.705Z";
            traits =             {
                anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
                email = "tool@fake-segment-tester.com";
                userId = "segment-fake-tester";
            };
            type = identify;
            userId = "segment-fake-tester";
        },
                {
            anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
            context =             {
                app =                 {
                    build = "1.0";
                    name = "Segment-Flurry_Example";
                    namespace = "org.cocoapods.demo.Segment-Flurry-Example";
                    version = "1.0";
                };
                device =                 {
                    id = "2B3659A2-DAD3-4E55-BC07-8A9D6E634CA3";
                    manufacturer = Apple;
                    model = "x86_64";
                    type = ios;
                };
                library =                 {
                    name = "analytics-ios";
                    version = "3.7.1";
                };
                locale = "en-US";
                network =                 {
                    cellular = 0;
                    wifi = 1;
                };
                os =                 {
                    name = iOS;
                    version = "13.4.1";
                };
                screen =                 {
                    height = 568;
                    width = 320;
                };
                timezone = "America/Los_Angeles";
                traits =                 {
                    anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
                    email = "tool@fake-segment-tester.com";
                    userId = "segment-fake-tester";
                };
            };
            event = "Completed Order";
            integrations =             {
            };
            messageId = "0190F4CD-44F6-4C22-8C7E-37197F25B896";
            properties =             {
                revenue = "14.51";
                title = "Launch Screen";
            };
            timestamp = "2020-04-16T18:58:23.706Z";
            type = track;
            userId = "segment-fake-tester";
        },
                {
            anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
            context =             {
                app =                 {
                    build = "1.0";
                    name = "Segment-Flurry_Example";
                    namespace = "org.cocoapods.demo.Segment-Flurry-Example";
                    version = "1.0";
                };
                device =                 {
                    id = "2B3659A2-DAD3-4E55-BC07-8A9D6E634CA3";
                    manufacturer = Apple;
                    model = "x86_64";
                    type = ios;
                };
                library =                 {
                    name = "analytics-ios";
                    version = "3.7.1";
                };
                locale = "en-US";
                network =                 {
                    cellular = 0;
                    wifi = 1;
                };
                os =                 {
                    name = iOS;
                    version = "13.4.1";
                };
                screen =                 {
                    height = 568;
                    width = 320;
                };
                timezone = "America/Los_Angeles";
                traits =                 {
                    anonymousId = "7955948E-DD94-42C4-8863-AD0E0BB1B587";
                    email = "tool@fake-segment-tester.com";
                    userId = "segment-fake-tester";
                };
            };
            integrations =             {
            };
            messageId = "05132F6A-AC89-4327-8052-4125B57A6572";
            name = Home;
            properties =             {
            };
            timestamp = "2020-04-16T18:58:23.706Z";
            type = screen;
            userId = "segment-fake-tester";
        }
    );
    sentAt = "2020-04-16T18:58:53.707Z";
}.
```

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SDK ?= "iphonesimulator"
-DESTINATION ?= "platform=iOS Simulator,name=iPhone 7"
+DESTINATION ?= "platform=iOS Simulator,name=iPhone 9"
```
Comment (briemcnally) [inline Makefile]: I don't think iPhone 9 is a thing. if you run `xcrun simctl list` you can get a list of devices and names. Maybe choose iPhone X if you want to bump the simulator to a newer one. The OS and the simulator don't always align.

**Review by briemcnally — APPROVED** [review]
LGTM other than the one comment.

File: Makefile (hunk 1 above)
Comment (omegabytes) [reply Makefile]: awesome thank you!

**Pull request merged**